testsuite/forge.py

```diff
@@ -942,33 +942,40 @@ def find_recent_images(
     num_images: int,
     # Set a generoush threshold in case of failures
     commit_threshold: int = 100,
+    enable_failpoints: bool = False,
```

Comment:
lets pass image_name here that decouples this logic from the intent of failpoints

Reply:
My thinking here was that it's easier to test if we have `enable_failpoints` and all these special cases in the function itself, since the logic might get more complex.

Reply:
I feel pretty strongly this is a violation of this interface, if you need to add more logic lets introduce it in another function and leave this function just searching for images, when we couple too much specific logic here it becomes unusable elsewhere. I probably should have passed in image_tag from the start :P